Networking/Networking/Model/ShippingLabel/Packages/Carriers and Rates/ShippingLabelCarriersAndRates.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+/// Represents the rate for a specific shipping carrier
```

Comment:
nit: This represents all of the available shipping carriers and rates (not a specific one), right?